Makefile

```diff
@@ -41,8 +44,14 @@ binary-local:
 build-container:
 	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)" .
 
+docs/%.1: docs/%.1.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+.PHONY: docs
+docs: $(MANPAGES_MD:%.md=%)
+
 clean:
-	rm -f skopeo
+	rm -f skopeo docs/*.1
 
 install: install-binary
 	install -m 644 man1/skopeo.1 ${MANINSTALLDIR}/man1/
```

Comment:
My commit for this in case you wanna checkout: https://github.com/lsm5/skopeo/commit/1f89e248d5c0b691ac7289e2863d9d319a967f36